### PR TITLE
Ch project setup

### PR DESCRIPTION
**Pull request by Tboy-AK**
## Description
Includes all necessary modules and folders in the `/src` folder

Finishes #3 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by kadirisunday** [comment]
code well documented

**Review by kadirisunday — DISMISSED** [review]

**Review by urbanishimwe — DISMISSED** [review]
LGTM!

File: src/index.js [hunk 1]
```diff
@@ -8,4 +8,4 @@ const startApp = async () => {
   header.textContent = programName;
 };
 
-document.addEventListener('DOMContentLoaded', startApp);
\ No newline at end of file
+document.addEventListener('DOMContentLoaded', startApp);
```
Comment (urbanishimwe) [inline src/index.js:11]: I don't this can be compilable by Node!

**Review by adebayoyeleye — APPROVED** [review]
as discussed

**Pull request merged**